### ISSUE TITLE
Wrong value of command property in dotnetBuild taskTemplate 

### ISSUE THREAD

**Issue by inouiw** [microsoft/vscode#78976]
- VSCode Version: 1.37.0
- OS Version: macOS 10.14.6

Steps to Reproduce:

1. Create .net core console app and open folder in vscode (`mkdir console1 && cd console1 && dotnet new console && code .`)
2. Message from C# (Extension) "Required assets to build and debug are missing from 'console2'. Add them?" --> Click "Not Now"
3. Click build task shortcut (Ctrl + Shift + b). Message says "No build task to run found. Configure Build Task..." --> Click on it
4. Message says "Create tasks.json file from template". --> Click on it
5. Message offers options. Click ".NET Core"
6. Click build task shortcut again. Then select "build". 

Error output is:
----------------------------------------------------------------
> Executing task: 'dotnet build' /property:GenerateFullPaths=true /consoleloggerparameters:NoSummary <

> /bin/bash: dotnet build: command not found
> The terminal process terminated with exit code: 127
----------------------------------------------------------------

Fix is to change file /src/vs/workbench/contrib/tasks/common/taskTemplates.ts
Change line with `command": "dotnet build"` to `command": "dotnet"` and add build to args array.

**Comment by inouiw** [comment]
@isidorn why did you add `*caused-by-extension` label? 
If the error is in this repository as I wrote in the bug report, then it should not be related to an extension. Also if you follow the steps to reporoduce and start vscosde with `--disable-extensions` then the error remains.

**Comment by inouiw** [comment]
Just tested, the problem does **not** occur on windows.

**Comment by isidorn** [comment]
dotnet is an extension.
So please file this here https://github.com/OmniSharp/omnisharp-vscode
Thanks!

**Comment by inouiw** [comment]
@isidorn as said the error also occurs if vscode is started with `--disable-extensions`. Also as said the error is in this repository. Here the full path and line:
`https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/tasks/common/taskTemplates.ts` line 30.

The line says `"command": "dotnet build"`. Obviously it is not supported on macOS if the argument is included in the command for that task.

Sorry that I need to correct you but `dotnet` is not a vscode extension. ".NET Core is a free and open-source, managed computer software framework for Windows, Linux, and macOS operating systems.".

**Comment by alexr00** [comment]
@akshita31 do you know if the template should be updated to have `build` as an argument?

**Comment by kinglionsz** [comment]
> @akshita31 do you know if the template should be updated to have `build` as an argument?

I noticed that issue (#78976) before I disclosed #79269. I thought it explained to me NOT clearly.It's no matter with the dotnet extensions.

and now I have updated VS CODE and build the task,there is no any build information in the terminal,why?
pls check.

tks. @alexr00
<img width="623" alt="tas with args error now ok or not" src="https://user-images.githubusercontent.com/51827261/63161004-ad118000-c051-11e9-9571-5cb634411694.png">

**Comment by inouiw** [comment]
My recommendation would be to delete everything in `https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/tasks/common/taskTemplates.ts` that can be provided by an extension. Only the externalCommand Template seems universally and that should remain. This would avoid code duplication (same code in this repository as in the extension repository). At the bottom of the file /taskTemplates.ts there are already some "Version 1.0 templates" commented out. 

Initially this issue caused a misunderstanding because of an error in an external tool (dotnet) which seemed not related to the vscode source. The issue only arose at my computer because I installed vscode on my mac, then created a dotnet core project which I opened in vscode and then I wanted to build and debug it. When opening the folder with the dotnet core solution, I was asked to install an extension. Which I did. However I did not wait until the extension was installed but immediately tried to build the project. So at that point the extension had not installed its own dotnet task template and the vscode template was used. If I would have known that I needed to wait for the extension then I would not have used the vscode template because this template is different then the one that is used for all projects that are opened after installing the extension.

So I would recommend to delete all task templates (dotnetCore, msbuild, maven) except externalCommand. Then I would test to install vscode and then open each of those project types. The expected behaviour would be that a popup appears to install the appropriate extension and that the code can then be build and debugged.
